### ISSUE TITLE
Write a BlogPost featuring cleanlab

### ISSUE THREAD

**Issue by jwmueller** [cleanlab/cleanlab#360]
Blog posts are extremely helpful to helping us explore new applications of cleanlab and introduce the tool to more data scientists.  You're of course free to publish your blogpost wherever you like (eg. Medium, Towards Data Science, reddit/learnmachinelearning, Kaggle forums, etc).  If interested, feel free to discuss ideas with us and get feedback in our community slack channel:  https://cleanlab.ai/slack  (can message Jonas Mueller if you wish to discuss 1:1)

An ideal blogpost will be a Jupyter notebook that highlights an interesting application of cleanlab on real or simulated data. 
Some ideas for topics include:

0) use cleanlab to find errors in a popular ML benchmark dataset. See: labelerrors.com for inspiration. There are many famous datasets which nobody has tried running through cleanlab yet. 

1) use cleanlab for multi-label classification task such as image or text tagging (where multiple labels can apply to each example).  If image tagging, we recommend using with pytorch-vision or timm package. If text tagging, we recommend using with huggingface package. But many other image/text modeling packages are also great to explore!

2) use cleanlab as part of a solution for a Kaggle competition, or MachineHack competition, or data-centric AI competition like [DataPerf](https://dataperf.org/challenges.html). Note many of these competitions have forums/notebook/kernel sections where you can publish your code to help others.

3) investigate which types of models as best for using with cleanlab to find (simulated) label errors in classification with tabular data (eg. XGBoost vs LightGBM vs CatBoost vs RandomForest vs NeuralNet vs SVM)

4) use cleanlab with NLP model to find label errors in interesting text classification dataset, such as classifying whether pairs of sentences are paraphrases or not, or whether an annotated question-answer pair is correct or not.

5) use cleanlab in less-standard ML task that is not exactly classification but similar, such as: metric learning (or contrastive learning), information retrieval, object detection, classification with data labeled by multiple annotators, etc.
